### PR TITLE
revise social links usage.

### LINKED ISSUE #15: Social Links
Hey,
could there be a way of setting the social links in a script block like this

``` html
<script>
var xyz = 'http://facebook.com/user';
</script>
```

Another thought would be to set additional info like the title and the service in an js object

``` html
<script>
var xyz = {
   service: fb,
   link:'http://face...',
   title: '',
   fa_icon: ''
};
</script>
```

### PR DESCRIPTION
**Pull request by 0t2**
Fixes #15
Facebook and Twitter links now grab from general setting, extra configuration as shown below: 
```javascript
// Code injection - Blog Header
<script>
    var socialConfig = {
        facebook: {
            title: "Caffeine Coding on Facebook"
        },
        twitter: {
            title: "@KelyvinN on Twitter"
        },
        linkedIn: {
            link: "https://linkedin.com/in/kelyvin",
            title: "Kelyvin on LinkedIn"
        },
        github: {
            link: "https://github.com/kelyvin",
            title: "Kelyvin on Github"
        }
    };
</script>
```

### CODE REVIEW

File: assets/js/src/cover.js [hunk 1]
```diff
@@ -98,7 +94,30 @@ $(function() {
...
+            target.attr("href", config.link);
+            replaceTitle(target, config.title);
+        }
+        else {
+            target.parent().hide(0);
+        }
+    }
+
+    function replaceTitle(target, title) {
+        console.log(title);
```
Comment (kelyvin) [inline assets/js/src/cover.js]: Can you remove this console.log?

**Comment by kelyvin** [comment]
HI @0t2, 

Sorry for the late reply, was out of town. Thanks for making these changes! Looks great!

I just made one minor comment. Can you also update the readme with the new configuration changes you've implemented? You can update the following section for [Social Networks]( https://github.com/kelyvin/caffeine-theme#social-networks)

Thanks!

File: assets/js/src/cover.js (hunk 1 above)
Comment (0t2) [reply assets/js/src/cover.js]: Sure!

**Comment by kelyvin** [comment]
Thanks for submitting this change! Looks great!

**Pull request merged**